Automattic/wp-calypso
# Help Center: Refactoring shouldUseHelpCenterExperience

## Proposed Changes

This PR cleans up all the machinery we used to set up the recent Support Experience experiment.
We are now at 100% with the new version, and so it's time to do some cleaning.

## Testing Instructions



- Test the new Support Experience:
     - Start a chat with Odie
     - Start a chat with an Happiness Engineer in staging
     - Use a free user to check that you are sent to forums
     - Check History (you need to have > 1 conversations)
     - Sanity check for articles
